internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1707,6 +1719,7 @@ func resourceKubernetesClusterRead(d *pluginsdk.ResourceData, meta interface{})
 			d.Set("private_link_enabled", accessProfile.EnablePrivateCluster)
 			d.Set("private_cluster_enabled", accessProfile.EnablePrivateCluster)
 			d.Set("private_cluster_public_fqdn_enabled", accessProfile.EnablePrivateClusterPublicFQDN)
+			d.Set("run_command_enabled", !(*accessProfile.DisableRunCommand))
```

Comment:
This might not always be returned from the API so could we nil check this and also set a default value, something like this:
```suggestion
			runCommandEnabled := true
			if *accessProfile.DisabledRunCommand != nil {
				runCommandEnabled = !*accessProfile.DisabledRunCommand
			}
			d.Set("run_command_enabled", runCommandEnabled)
```